extensions/amp-story/1.0/progress-bar.js

```diff
@@ -35,14 +37,41 @@ const TRANSITION_LINEAR = `transform ${POLL_INTERVAL_MS}ms linear`;
  */
 const TRANSITION_EASE = 'transform 200ms ease';
 
+/**
+ * Size in pixels of a segment ellipse.
+ * @const {number}
+ */
+const ELLIPSE_WIDTH_PX = 3;
+
+/**
+ * Size in pixels of the total side margins of a segment.
+ * @const {number}
+ */
+const SEGMENTS_MARGIN_PX = 4;
```

Comment:
Can we try a smaller margin for the ellipses and see if it looks better? It's hard to tell from the mocks.